includes/utils.php

```diff
@@ -302,47 +316,11 @@ function distributable_post_types() {
 	 * @hook distributable_post_types
 	 *
 	 * @param {array} Post types that are distributable. Default all post types except `dt_ext_connection` and `dt_subscription`.
+	 * @param {string} The type of output to return. Accepts post type 'names' or 'objects'. Default 'names'.
 	 *
 	 * @return {array} Post types that are distributable.
 	 */
-	return apply_filters( 'distributable_post_types', array_diff( $post_types, [ 'dt_ext_connection', 'dt_subscription' ] ) );
-}
-
-/**
- * Return post types that should be excluded from the permission list.
- *
- * @since  1.7.0
- * @return array
- */
-function get_excluded_post_types_from_permission_list() {
-	// Hide the built-in post types except 'post' and 'page'.
-	$hide_from_list = get_post_types(
-		array(
-			'_builtin'     => true,
-			'show_in_rest' => true,
-		)
-	);
-	unset( $hide_from_list['post'], $hide_from_list['page'] );
-
-	// Default is keyed by the post type 'post' => 'post', etc; hence using `array_values`.
-	$hide_from_list = array_values( $hide_from_list );
-
-	/**
-	 * Filter to update the list of post types that should be hidden from the "Post types permissions" list.
-	 *
-	 * @since 1.7.0
-	 * @hook dt_excluded_post_types_from_permission_list
-	 *
-	 * @param {array} The list of hidden post types.
-	 *
-	 * @return {bool} The updated array with the list of post types that should be hidden.
-	 */
-	$hide_from_list = apply_filters( 'dt_excluded_post_types_from_permission_list', $hide_from_list );
-
-	// Strict Hide 'dt_subscription' post type.
-	$hide_from_list[] = 'dt_subscription';
-
-	return $hide_from_list;
+	return apply_filters( 'distributable_post_types', $post_types, $output );
```

Comment:
```suggestion
	$post_types = apply_filters( 'distributable_post_types', $post_types );

	if ( 'names' === $output ) {
		return $post_types;
	}

	return array_map( 'get_post_type_object', $post_types );
```

Per note on docblock